<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDikfkSwitchWindow</class>
 <widget class="QMainWindow" name="DDikfkSwitchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>229</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>229</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>231</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IK FK Switch Window</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>IK   FK   Switch</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <widget class="QComboBox" name="CharacterComboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>28</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshCharacterBTN">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>27</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>27</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>9</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_A">
              <property name="maximumSize">
               <size>
                <width>13</width>
                <height>13</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_A">
              <property name="styleSheet">
               <string notr="true">QGroupBox{
border:0px solid;
}
QGroupBox:enabled{
border:1px solid;
border-radius:6px;
border-top-color:qlineargradient(spread:pad, x1:0.1, y1:0.5, x2:0.9, y2:0.5, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
border-bottom-color:qlineargradient(spread:pad, x1:0.1, y1:0.5, x2:0.9, y2:0.5, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
border-left-color:qlineargradient(spread:pad, x1:0.5, y1:0.1, x2:0.5, y2:0.9, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
border-right-color:qlineargradient(spread:pad, x1:0.5, y1:0.1, x2:0.5, y2:0.9, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
}
</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="leftArmBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Left  Arm</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="rightArmBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Right  Arm</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="lefLegBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Left   Leg</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="rightLegBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Right   Leg</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_B">
              <property name="maximumSize">
               <size>
                <width>13</width>
                <height>13</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_B">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox{
border:0px solid;
}
QGroupBox:enabled{
border:1px solid;
border-radius:6px;
border-top-color:qlineargradient(spread:pad, x1:0.1, y1:0.5, x2:0.9, y2:0.5, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
border-bottom-color:qlineargradient(spread:pad, x1:0.1, y1:0.5, x2:0.9, y2:0.5, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
border-left-color:qlineargradient(spread:pad, x1:0.5, y1:0.1, x2:0.5, y2:0.9, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
border-right-color:qlineargradient(spread:pad, x1:0.5, y1:0.1, x2:0.5, y2:0.9, stop:0.1 rgba(170, 255, 127, 255), stop:0.2 rgba(170, 255, 127, 0), stop:0.8 rgba(170, 255, 127, 0), stop:0.943503 rgba(170, 255, 127, 255));
}
</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>2</number>
               </property>
               <property name="margin">
                <number>3</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="leftForeLegBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Left  ForeLeg</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="rightForeLegBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Right  ForeLeg</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="leftHindLegBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Left   HindLeg</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="rightHindLegBTN">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>28</height>
                    </size>
                   </property>
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true"/>
                   </property>
                   <property name="text">
                    <string>Right  HindLeg</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionLeftArmSwitch">
   <property name="text">
    <string>leftArmSwitch</string>
   </property>
  </action>
  <action name="actionRightArmSwitch">
   <property name="text">
    <string>RightArmSwitch</string>
   </property>
  </action>
  <action name="actionLeftLegSwitch">
   <property name="text">
    <string>LeftLegSwitch</string>
   </property>
  </action>
  <action name="actionRightLegSwitch">
   <property name="text">
    <string>RightLegSwitch</string>
   </property>
  </action>
  <action name="actionLeftForeLegSwitch">
   <property name="text">
    <string>leftForeLegSwitch</string>
   </property>
  </action>
  <action name="actionRightForeLegSwitch">
   <property name="text">
    <string>rightForeLegSwitch</string>
   </property>
  </action>
  <action name="actionLeftHindLegSwitch">
   <property name="text">
    <string>leftHindLegSwitch</string>
   </property>
  </action>
  <action name="actionRightHindLegSwitch">
   <property name="text">
    <string>rightHindLegSwitch</string>
   </property>
  </action>
  <action name="actionRefreshCharacter">
   <property name="text">
    <string>refreshCharacter</string>
   </property>
  </action>
  <action name="actionCurrentCharacterChanged">
   <property name="text">
    <string>currentCharacterChanged</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>leftForeLegBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionLeftForeLegSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightLegBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionRightLegSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightHindLegBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionRightHindLegSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightForeLegBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionRightForeLegSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftHindLegBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionLeftHindLegSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rightArmBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionRightArmSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lefLegBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionLeftLegSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leftArmBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionLeftArmSwitch</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CharacterComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>actionCurrentCharacterChanged</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_B</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_B</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_A</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_A</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refreshCharacterBTN</sender>
   <signal>clicked()</signal>
   <receiver>actionRefreshCharacter</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
